php/page.php

```diff
@@ -1,5 +1,4 @@
 <?php
-$templateBasePath = "template/";
```

Comment:
Intentional? The reason for this is to eventually allow different site templates - so it'd look like `template/template1/index.html`. At the moment, everything is directly in `template/` though

Reply:
Ahha, that makes sense. I moved it to global.php but I think knowing this helps me understand that it's okay to just copy that value out and put it directly into the install code.